k-distribution/tests/pyk/emit-json-spec-tests/emit_json_spec_test.py

```diff
@@ -73,6 +73,17 @@ def test_prove(self):
         # Then
         self.assertEqual(result['label'], '#Top')
 
+    def test_prove_claim_with_lemmas(self):
```

Comment:
This test could go into its own module (including a separate `verification.k` file), what do you think?

(Perhaps we should also extract a superclass `PykTestCase` that takes care of the initialization of the KProve instance.)

Reply:
I will make these updates after this one goes through: https://github.com/runtimeverification/k/pull/2438